{{cookiecutter.project_name}}/docs/template_usage.md

```diff
@@ -142,78 +142,99 @@ Once authorized, pre-commit.ci should automatically be activated.
-    -   [flake8-tidy-imports](https://github.com/adamchainz/flake8-tidy-imports):
-        tidy module imports.
-    -   [flake8-docstrings](https://github.com/PyCQA/flake8-docstrings):
-        pydocstyle extension of flake8.
-    -   [flake8-rst-docstrings](https://github.com/peterjc/e8-rst-docstrings):
-        extension of `flake8-docstrings` for `rst` docs.
-    -   [flake8-comprehensions](https://github.com/adamchainz/e8-comprehensions):
-        write better list/set/dict comprehensions.
-    -   [flake8-bugbear](https://github.com/PyCQA/flake8-bugbear):
-        find possible bugs and design issues in program.
-    -   [flake8-blind-except](https://github.com/elijahandrews/flake8-blind-except):
-        checks for blind, catch-all `except` statements.
--   [yesqa](https://github.com/asottile/yesqa):
-    remove unneccesary `# noqa` comments, follows additional dependencies listed above.
--   [autoflake](https://github.com/PyCQA/autoflake):
-    remove unused imports and variables.
--   [pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks): generic pre-commit hooks.
+The following pre-commit hooks are for code style and format:
+
+-   [black](https://black.readthedocs.io/en/stable/):
+    standard code formatter in Python.
+-   [blacken-docs](https://github.com/asottile/blacken-docs):
+    black on Python code in docs.
+-   [prettier](https://prettier.io/docs/en/index.html):
+    standard code formatter for non-Python files (e.g. YAML).
+
+The following pre-commit hooks are for errors and inconsistencies:
+
+-   [ruff][]: Many configurable checks for Python code, see [Overview of Ruff checks](#overview-of-ruff-checks).
+-   [pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks): generic pre-commit hooks for text files.
     -   **detect-private-key**: checks for the existence of private keys.
     -   **check-ast**: check whether files parse as valid python.
-    -   **end-of-file-fixer**:check files end in a newline and only a newline.
+    -   **end-of-file-fixer**: check files end in a newline and only a newline.
     -   **mixed-line-ending**: checks mixed line ending.
     -   **trailing-whitespace**: trims trailing whitespace.
     -   **check-case-conflict**: check files that would conflict with case-insensitive file systems.
--   [pyupgrade](https://github.com/asottile/pyupgrade):
+-   **forbid-to-commit**: Make sure that `*.rej` files cannot be commited.
+    These files are created by the [automated template sync](#automated-template-sync)
+    if there's a merge conflict and need to be addressed manually.
+
+[ruff]: https://beta.ruff.rs/docs/
+
+#### Overview of Ruff checks
+
+[Ruff][] implements several checks for Python code and docstrings.
+Therefore each check below mentions a the rule prefix that enables it when added to `pyproject.toml`.
+E.g. `isort` is enabled with:
+
+```toml
+[tool.ruff]
+select = [..., "I", ...]
+```
```

Comment:
It looks like something around here is messing up github's parsing of the file. Any idea what's up with that?

Reply:
Feels wrong to change the way you write documentation to appease GitHub’s syntax highlighter, but this should do the trick.